### PR TITLE
Task general

### PR DESCRIPTION
**Pull request by Parsley-Afroman**
This is my overworked general layout that has content you can ignore if you want to assess the layout.

Please ignore the About & Portfolio Section.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html>
```
Comment (luke-grayland) [inline index.html:2]: Could you add lang="en" to the html tag for accessibility purposes?

File: index.html [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+				<div id="logo-wrap" class="wrap">
+					<img src="#" alt="logo"/>
+				</div>
+				<div id="hi-wrap" class="wrap">
+					<h2>Hey Hey</h2>
+					<h3>Hope You Enjoy</h3>
+					<p>This is some brief text introducing you to the page</p>
+				</div>
+			</div>	
+
```
Comment (luke-grayland) [inline index.html:30]: Could you cut out some of the white space here to improve readability?

File: index.html [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+			<div id="proj-C" class="proj-wrap">
+				<h4>Project Name</h4>
+				<p>Some Brief Text About the Project</p>
+				<img src="#" alt="project-image-C"/>
+				<a href="#" class="git-link"></a>
+				<a href="#" class="live-link"></a>
+			</div>
+			
+		</section>
+		<footer>
```
Comment (luke-grayland) [inline index.html:93]: Could you make the footer 'display: flex' and 'justify-content: space between' in CSS to spread the elements out horizontally?

File: index.html [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+	<head>
+		<meta charset="UTF-8">
+		<link href="./normalize.css" type="text/css"  rel="stylesheet">
+		<link href="./style.css" type="text/css" rel="stylesheet">
+		<meta name="viewport" content="width=device-width">
+	</head>
+	<body>
+		
+		<nav>
+			<button>About</button>
```
Comment (luke-grayland) [inline index.html:12]: Now that you've got these sections in the layout, could you link these buttons up with href="#id"?

File: style.css [hunk 5]
```diff
@@ -0,0 +1,139 @@
+html {
+	font-size: 62.5%;
+}
+
+* {
+	box-sizing: border-box;
+}
+html {
```
Comment (luke-grayland) [inline style.css:8]: Is this a duplicate of the html selector above?

**Review by luke-grayland — COMMENTED** [review]
Looking good! 

When I scroll far to the right I get a big white blank space. Could you play around with the width to try and resolve this?

File: style.css (hunk 5 above)
Comment (Parsley-Afroman) [reply style.css:8]: Yes it was and I have removed it

File: index.html (hunk 4 above)
Comment (Parsley-Afroman) [reply index.html:12]: Good shout, I will definitely link them up; however, I was thinking of just making them anchors as I only used the button as I thought we needed to out of accessibility, but evidently, from Mike's page, we don't.

File: index.html (hunk 3 above)
Comment (Parsley-Afroman) [reply index.html:93]: Sure thing

File: style.css [hunk 6]
```diff
@@ -0,0 +1,179 @@
...
+	display: flex;
+}
+
+@media only screen and (max-width: 600px){
+	#nav-list {
+		display: none;
+	}
+}
+
+#nav-list li {
```
Comment (luke-grayland) [inline style.css:29]: Could you use a class here instead of an ID in case you ever need to use this selector again?

**Review by luke-grayland — COMMENTED** [review]
Structure is nice and tidy. Looks like this will be a much easier base to build from. The only questions I have are around the use of ID's where classes could be used.

**Pull request merged**